etcdctl/READMEv2.md

```diff
@@ -291,10 +291,7 @@ The following exit codes can be returned from etcdctl:
 
 ## Endpoint
 
-If your etcd cluster isn't available on `http://127.0.0.1:2379` you can specify
-a `--endpoint` flag or `ETCDCTL_ENDPOINT` environment variable. You can list one endpoint,
-or a comma-separated list of endpoints. This option is ignored if the `--discovery-srv`
-option is provided.
+If the etcd cluster isn't available on `http://127.0.0.1:2379`, specify a `--endpoint` flag or `ETCDCTL_ENDPOINT` environment variable. One endpoint or a comma-separated list of endpoints can be listed. This option is ignored if the `--discovery-srv` option is provided.
```

Comment:
the etcdctl v2 says to use `--endpoints` since `--endpoint` is deprecated.  Should we use `--endpoints` there?

```
--endpoint value                 DEPRECATED - "--endpoints" should be used instead
--endpoints value                a comma-delimited list of machine addresses in the cluster (default: "http://127.0.0.1:2379,http://127.0.0.1:4001")
```